### PR TITLE
scripts/gen_tee_bin.py: convert symbol and section name to strings

### PR DESCRIPTION
**Pull request by jforissier**
Older versions of pyelftools return symbol and section names as byte
arrays rather than plain strings. As a result, symbols are not found:

 $ make
   ...
   GEN     out/arm/core/tee-header_v2.bin
 Cannot find symbol _start
 core/arch/arm/kernel/link.mk:191: recipe for target 'out/arm/core/tee-header_v2.bin' failed
 make[1]: *** [out/arm/core/tee-header_v2.bin] Error 1

(This error was observed when building on Ubuntu 16.04.)

Signed-off-by: Jerome Forissier <jerome@forissier.org>

### CODE REVIEW

File: scripts/gen_tee_bin.py [hunk 1]
```diff
@@ -72,15 +72,20 @@ def get_symbol(elffile, name):
                          if isinstance(s, SymbolTableSection)]
         for section in symbol_tables:
             for symbol in section.iter_symbols():
+                # .name might be a byte array or a string, we want a string
```
Comment (jenswi-linaro) [inline scripts/gen_tee_bin.py]: How about adding a helper function for this? `get_symbol_name()` for instance.

**Comment by jenswi-linaro** [comment]
Nice catch by the way.

File: scripts/gen_tee_bin.py (hunk 1 above)
Comment (jforissier) [reply scripts/gen_tee_bin.py]: Sure.

**Comment by jforissier** [comment]
Update

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**